src/convertToObject.js

```diff
@@ -3,16 +3,36 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles (see an example
+ * in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of
+ * related CSS properties (see an example
+ * in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stringCleared = sourceString.replace(/ /g, '').split('\n').join('');
+  const finalString = stringCleared
+    .replaceAll('1pxsolid#e8e8e8', '1px solid #e8e8e8')
+    .replaceAll('left!important', 'left !important')
+    .replaceAll('all.2sease-in-out', 'all .2s ease-in-out');
```

Comment:
the tests throw an error because they don't understand what replaceAll is (since this method was added very recently)

so let's replace it with a plain `replace` using the g (global) flag.

example: 
```
 .replace(/1pxsolid#e8e8e8/g, '1px solid #e8e8e8')
```